docs/source/features/stablehlo.md

```diff
@@ -3,172 +3,67 @@
 This document describes how to use torch export + torch xla to export to
 [StableHLO](https://github.com/openxla/stablehlo) format.
 
+There are 2 ways to accomplish this:
+
+1. First do torch.export to create a ExportedProgram, which contains the program
+   in torch.fx graph. Then use `exported_program_to_stablehlo` to convert it into an object that contains 
+   stablehlo MLIR code.
+2. First convert pytorch model to a jax function, then use jax utilities to convert it
+   to stablehlo
+
+## Using `torch.export`
+
 ``` python
 from torch.export import export
-from torch_xla.stablehlo import exported_program_to_stablehlo
-import torch_xla.core.xla_model as xm
 import torchvision
 import torch
-
-xla_device = xm.xla_device()
+import torch_xla2 as tx
+import torch_xla2.export
 
 resnet18 = torchvision.models.resnet18()
 # Sample input is a tuple
 sample_input = (torch.randn(4, 3, 224, 224), )
 output = resnet18(*sample_input)
 exported = export(resnet18, sample_input)
-stablehlo_program = exported_program_to_stablehlo(exported)
-
-# Now stablehlo_program is a callable backed by stablehlo IR.
-
-# we can see it's stablehlo code with
-#   here 'forward' is the name of function. Currently we only support
-#   one entry point per program, but in the future we will support
-#   multiple entry points in a program.
-print(stablehlo_program.get_stablehlo_text('forward'))
-
-# we can also print out the bytecode
-print(stablehlo_program.get_stablehlo_bytecode('forward'))
-
-# we can also run the module, to run the stablehlo module, we need to move 
-# our tensors to XLA device.
-sample_input_xla = tuple(s.to(xla_device) for s in sample_input)
-
-output2 = stablehlo_program(*sample_input_xla) 
-print(torch.allclose(output, output2.cpu(), atol=1e-5)) 
-```
-
-## Saving StableHLO bytecodes to disk
-
-One can now save stablehlo to disk with
-
-``` python
-stablehlo_program.save('/tmp/stablehlo_dir')
-```
-
-The path should be path to an empty directory. If it doesn't exist, it
-will be created. This directory can be loaded again as another
-stablehlo_program:
-
-``` python
-from torch_xla.stablehlo import StableHLOGraphModule
-stablehlo_program2 = StableHLOGraphModule.load('/tmp/stablehlo_dir')
-output3 = stablehlo_program2(*sample_input_xla) 
-```
-
-## Convert saved StableHLO for serving
```

Comment:
Is there a reason to remove this section?

Reply:
tf.serving is kind of outdated. So we don't want to actively recommend it anymore.

Reply:
Ack